wire-ios/Wire-iOS/Sources/UserInterface/Company Login/CompanyLoginController.swift

```diff
@@ -330,9 +330,14 @@ extension CompanyLoginController {
             case let .success(backendEnvironment):
                 requestUserConfirmationForBackendSwitch(to: backendEnvironment) { didConfirm in
                     guard didConfirm else { return }
-                    sessionManager.switchBackend(to: backendEnvironment)
-                    BackendEnvironment.shared = backendEnvironment
-                    self.startAutomaticSSOFlow(promptOnError: false)
+                    sessionManager.switchBackend(to: backendEnvironment) { error in
```

Comment:
swift lint is confused here... I am using the `error`.

Reply:
Very weird